crates/primitives/src/identity.rs

```diff
@@ -90,6 +90,10 @@ pub enum WalletType {
         #[serde(rename = "chainId")]
         chain_id: u64,
     },
+    SN {
```

Comment:
Let's use a full name to make our life easier :)

Reply:
Updated to full name